### DISCUSSION TITLE
Welcome to HCX Discussions!

### DISCUSSION THREAD

**Discussion by howeecross** [howeecross/HCX#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hcoinx** [comment]
Everything is going well.

**Comment by hcoinx** [comment]
![163077851-a3f98c75-52e5-4517-84ff-4f8b95c0b2dd](https://user-images.githubusercontent.com/76787189/172740090-f7668786-e349-4c8a-b1d9-9020ad366d69.jpg)
